src/Client/Resource.php

```diff
@@ -15,9 +13,9 @@
     const FETCH_PROMISE = 'promise';
 
     /**
-     * @var HttpClient|HttpAsyncClient
+     * @var FlexibleHttpClient
      */
-    protected $httpAsyncClient;
+    protected httpClient;
```

Comment:
`$` is missing 👁

Reply:
MOAR $ plzzzz

Reply:
lol, oups. Edit in place in github is not really the way to go!

Reply:
Fixed. Good catch.